src/renderer/editor/selection.ts

```diff
@@ -26,3 +27,30 @@ export const selectAllWords: () => void = () => {
     })
   );
 };
+
+export const selectSentence: () => void = () => {
+  const { currentProject } = store.getState();
+
+  if (currentProject === null || currentProject?.transcription === null) {
+    return;
+  }
+  let { startIndex, endIndex } = store.getState().selection.self;
+  endIndex -= 1; // Account for the fact that the selection is exclusive of the end index
+  startIndex -= 1; // Move past the current word to account for the edge case of selecting a sentence end word
+  let startWord = currentProject.transcription.words[startIndex];
+  while (!(checkSentenceEnd(startWord) && startWord.deleted === false)) {
```

Comment:
Clearer as `while (!checkSentenceEnd(startWord) || startWord.deleted) {`

Reply:
Should be fixed now